### DISCUSSION TITLE
process didn't exit successfully: `target\debug\examples\sqlite.exe` (exit code: 0xc0000135, STATUS_DLL_NOT_FOUND)

### DISCUSSION THREAD

**Discussion by adminSxs** [rusqlite/rusqlite#1103] [Questions] [answered]
```
OS: Windows10
use vcpkage: install dlib[sqlite3]
```
when i copy the example to run,result is `process didn't exit successfully: `target\debug\examples\sqlite.exe` (exit code: 0xc0000135, STATUS_DLL_NOT_FOUND)`

**Comment by gwenn** [comment, marked as answer]
https://stackoverflow.com/questions/7378959/how-to-check-for-dll-dependency ?
